src/MovieDetail/MovieDetail.css

```diff
@@ -0,0 +1,93 @@
+.movie-detail-wrapper {
+  /* background-color: #324653; */
+  height: 100%;
+  margin: 0;
+  font-family: 'Farro', sans-serif;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+
+.fas {
+  color: white;
+  margin-left: 5%;
+  margin-top: 3%;
+  width: 500px;
+}
```

Comment:
We might want to add a hover state for the cursor to indicate that the arrow is a button

Reply:
Good idea! I added this to the poster images in the Movie component as well to indicate that the posters are clickable to get more info on the movie.